src/hooks/useFeatureFlags.ts

```diff
@@ -0,0 +1,18 @@
+import { useLocation } from "react-router-dom"
+
+type FeatureFlags = {
+  useTags: boolean
+}
+
+function useQuery() {
```

Comment:
Can it be made `const` with arrow function?

Reply:
Fixed